app/models/event.rb

```diff
@@ -22,6 +22,14 @@ def self.filters_to_json
     ].to_json
   end
 
+  def self.years_to_json
+    events = Event.all.order(start_date: :asc)
```

Comment:
If you are ordering in the two lines below, you don't need to sort here as well.

Reply:
Thanks @Levia I will fix this!